serverless_plugins/lambda-update-deprecated-runtime.js

```diff
@@ -0,0 +1,29 @@
+class LambdaUpdateDeprecatedRuntime {
+    constructor(serverless, options) {
+        this.serverless = serverless;
+        this.options = options;
+        this.provider = 'aws';
+
+        this.hooks = {
+            'before:deploy:deploy': this.beforeDeploy.bind(this),
+        };
+    }
+
+    beforeDeploy() {
```

Comment:
Never seen a custom serverless plugin like this. Did you find this from a resource?

Reply:
It is inspired by this https://github.com/serverless/serverless/issues/11337 then I had to adapt to the severless version 3, which have different hook names see serverless docu https://www.serverless.com/framework/docs/guides/plugins/creating-plugins